service/history/historyEngine.go

```diff
@@ -2850,11 +2850,11 @@ func getScheduleID(
 ) (int64, error) {
 
 	if activityID == "" {
-		return 0, serviceerror.NewInvalidArgument("Neither ActivityID nor ScheduleID is provided")
+		return 0, serviceerror.NewInvalidArgument("invalid ActivityID provided")
```

Comment:
`Invalid activity ID` ?

Reply:
Okay, that is a fair point.
idiomatic go says errors should be lower cased so they can be `fmt'ed` into a logging line which may be sentenced cased.

I did depart from what was there before, happy to change though to keep consistency.

Thoughts?

Reply:
I would just say `activityID cannot be empty`